cli/azd/pkg/tools/git/git.go

```diff
@@ -143,6 +143,13 @@ func (cli *gitCli) InitRepo(ctx context.Context, repositoryPath string) error {
 		return fmt.Errorf("failed to init repository: %s: %w", res.String(), err)
 	}
 
+	// Set initial branch to main
+	runArgs = exec.NewRunArgs("git", "-C", repositoryPath, "branch", "-M", "main")
```

Comment:
Might be faster to just pass `-b` to specify the initial branch in `git init`

Reply:
older versions of git don't support `git init -b` Versions of git prior to 2.28.0 don't support -b 
https://git-scm.com/docs/git-init/2.27.0

This change is specifically needed for older versions of git that default to the master branch instead of main. Newer versions of git default to main and this is no-op.